numpy/_core/tests/test_multiarray.py

```diff
@@ -388,12 +396,16 @@ def set_strides(arr, strides):
                                                     shape=(10,), strides=(-1,))
         assert_raises(ValueError, set_strides, x[::-1], -1)
         a = x[::-1]
-        a.strides = 1
-        a[::2].strides = 2
+
+        with warnings.catch_warnings():  # gh-28901
+            warnings.filterwarnings(action="ignore", category=DeprecationWarning)
+
+            a.strides = 1
+            a[::2].strides = 2
 
         # test 0d
         arr_0d = np.array(0)
-        arr_0d.strides = ()
+        arr_0d = np.lib.stride_tricks.as_strided(arr_0d, strides = ())
```

Comment:
Remove spaces around `=`  for arguments to functions (PEP8) -- also in the other cases below.